File: 장주성/20230515/프로세스.py
def solution(priorities, location):
    answer = 0
    arr = []
    max_num = max(priorities)
    while max_num != 0:
        for i, v in enumerate(priorities):
            if v == max_num:
                arr.append(v)
                priorities[i] = 0
            else:
                continue
    print(arr)
    print(priorities)

    return answer

File: 이수창/20230515/프로세스.js
function solution(priorities, location) {
    var answer = 0;
    let print = 0
    while (true) {
        var first = priorities.shift()
        if (priorities.filter((prior) => prior > first).length > 0) {
            priorities.push(first)
        } 
        else {
            print ++
            if (location == 0) {
                return answer = print
            }
        }
    
        location --
       
        if (location === -1) {
            
            location = priorities.length - 1
        }
}